pkgs/development/compilers/crystal/default.nix

```diff
@@ -27,6 +27,8 @@
   tzdata,
   which,
   zlib,
+  # https://github.com/crystal-lang/crystal/issues/12299
+  withRepl ? stdenv.hostPlatform.isx86_64,
```

Comment:
Feel free to suggest a better name for `withRepl`. While I only considered REPL use cases, the upstream commit log mentions other uses. :bow:

https://github.com/crystal-lang/crystal/commit/c2be6220b1745a5948e8fff90ab96a30a1902c56